package.json

```diff
@@ -99,6 +99,7 @@
     "proxy": "^1.0.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "rimraf": "^3.0.2",
```

Comment:
Looks like we already have it https://github.com/microsoft/playwright/blob/4f0103fef0e70000ee89bf7503891d57e805e6b9/packages/playwright-test/package.json#L42

Reply:
Cosider using rimrafBundle here?

Reply:
Yes, it is already using bundle in the src, was a stray dependency.